spooky-doodle/Bangazon-Platform-API
# Implemented computers endpoint.   **Breaking Change**

# Description

Implemented Computers endpoint.  GET, POST, PUT, and DELETE all work.  DELETE is blocked if the `ComputerEmployee` table has an active employee assignment.

**BREAKING CHANGE**  Fixed typo in database schema. Computer endpoint had a `DecomissionDate` table entry, I corrected spelling by renaming it to `DecommissionDate`.

Fixes #9, #21 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1.  DROP Computer table from testing and live database.
2.  Run `testing_bangazon.sql` to refresh database.
3.  Run `bangazon.sql` to regenerate computer table.
4.  All tests should now pass.



# Checklist:
